content/en/news/releases/1.8.x/announcing-1.8/change-notes/index.md

```diff
@@ -0,0 +1,172 @@
+default, can be disabled by `REQUIRE_3P_TOKEN` environment variable in Istiod, which will require new tokens with audience. The
+`TOKEN_AUDIENCES` environment variable allows customizing the checked audience, default remains `istio-ca`.
+
+- **Added** `AuthorizationPolicy` now supports a `Source` of type `remoteIpBlocks`/`notRemoteIpBlocks` that map to a new `Condition` attribute called `remote.ip` that can also be used in the "when" clause.  If using an http/https load balancer in front of the ingress gateway, the `remote.ip` attribute is set to the original client IP address determined by the `X-Forwarded-For` http header from the trusted proxy configured through the `numTrustedProxies` field of the `gatewayTopology` under the `meshConfig` when you install Istio or set it via an annotation on the ingress gateway.  See the documentation here: [Configuring Gateway Network Topology](/docs/ops/configuration/traffic-management/network-topologies/). If using a TCP load balancer with the Proxy Protocol in front of the ingress gateway, the `remote.ip` is set to the original client IP address as given by the Proxy Protocol.
+ ([reference](/docs/reference/config/security/authorization-policy/))([usage](/docs/ops/configuration/traffic-management/network-topologies/))([usage](/docs/tasks/security/authorization/authz-ingress/)) ([Issue #22341](https://github.com/istio/istio/issues/22341))
+
+- **Added** Trust Domain Validation by default rejecting requests in sidecars if the request is not from same trust domain
+or if it's not in the `TrustDomainAliases` specified in the `MeshConfig`.
+  ([Issue #26224](https://github.com/istio/istio/issues/26224))
+
+## Telemetry
+
+- **Updated** the "Control Plane Dashboard" and the "Performance Dashboard" to use the `container_memory_working_set_bytes` metric
+to display memory. This metric only counts memory that *cannot be reclaimed* by the kernel even under memory pressure,
+and therefore more relevant for tracking. It is also consistent with `kubectl top`. The reported values are lower than
+the previous values.
+
+- **Added** Update Istio Workload and Istio Service dashboards to improve loading time.
+  ([Issue #22408](https://github.com/istio/istio/issues/22408))
+
+- **Added** `datasource` parameter to Grafana dashboards
+  ([Issue #22408](https://github.com/istio/istio/issues/22408))
+
+- **Added** Listener Access Logs when `ResponseFlag` from Envoy is set.
+  ([Issue #26851](https://github.com/istio/istio/issues/26851))
+
+- **Added** support for `OpenCensusAgent` formatted trace export with configurable trace context headers.
+
+- **Added** Proxy config to control Envoy native stats generation.
+  ([Issue #26546](https://github.com/istio/istio/issues/26546))
+
+- **Added** Istio Wasm Extension Grafana Dashboard.
+  ([Issue #25843](https://github.com/istio/istio/issues/25843))
```

Comment:
**Added** gRPC streaming message count proxy Prometheus metrics `istio_request_messages_total` and `istio_response_messages_total` ([PR](https://github.com/istio/proxy/pull/3048)).

Reply:
Yes. This needs to be added here.

@bianpengyuan @brian-avery we need a way to track and include changes that happen in istio/proxy with the same tooling. There are impactful changes that we may miss without that.

Reply:
@kyessenov can you update https://preliminary.istio.io/latest/docs/reference/config/metrics/ as well ?

Reply:
added @kyessenov 's additional release note